common/religion/doctrines/wc_doctrines_special.txt

```diff
@@ -212,3 +212,19 @@ doctrine_legions_puppet = {
 		}
 	}
 }
+doctrine_loyal_pawns = {
+	group = "special"
+	
+	is_available_on_create = {
+		has_doctrine = special_doctrine_loyal_pawns # Keeps on creating a new faith
+	}
+	
+	special_doctrine_loyal_pawns = {
+		parameters = {
+			is_in_dark_religion_family_trigger = yes
+		}
+		character_modifier = {
+			mercenary_count_mult = -1
+		}
+	}
+}
```

Comment:
It's not right. You just add the modifier to `doctrine_end_justifies_means`.

Reply:
Oh ok